simple-social-icons.php

```diff
@@ -353,8 +361,8 @@ function update( $newinstance, $oldinstance ) {
 				$newinstance[ $key ] = 0;
 			}
 
-			/** Accept empty colors */
-			elseif ( strpos( $key, '_color' ) && '' == trim( $value ) ) {
+			/** Accept empty colors for permitted keys. */
```

Comment:
Hey, @nickcernis Inline comments shouldn't use `//` instead of `/**`? I'm not completely sure which is the correct one.

Reply:
@marksabbath Yep, they should use the double-slash format (https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#5-1-single-line-comments). I didn't update the full file to avoid turning this into a full standards review (there are a bunch of other standards things that need fixing), but I'll make a separate commit for this to keep it consistent as you say.